src/map/mob.cpp

```diff
@@ -1243,7 +1243,11 @@ static int32 mob_can_changetarget(struct mob_data* md, struct block_list* target
 		case MSS_BERSERK:
 			if (!(mode&MD_CHANGETARGETMELEE))
 				return 0;
-			if (!(battle_config.mob_ai&0x80) && md->norm_attacked_id != target->id)
+			// If the special normal attacked event occured, always change target in berserk state
+			if (md->norm_attacked_id == target->id)
+				return 1;
+			// If the special setting to switch target even on skills is set, we need to verify the range here
```

Comment:
id change the comment to something like `0x80 is switch target on skills`
alternatively you could give 0x80 a better name and we could do away with the comment altogether

Reply:
I always used this style of describing it, but maybe I shouldn't even mention the setting - main purpose of this comment is to explain why we check for the distance here again in the code below.

I really don't want to start explaining what each hex value in mob_ai means everywhere we use it.

Giving each value in mob_ai a name should be a separate PR.